### DISCUSSION TITLE
chat example says naïve implementation

### DISCUSSION THREAD

**Discussion by rokit** [actix/examples#565] [Q&A] [answered]
https://github.com/actix/examples/blob/6c39b4b13882628b9e3649d9a69b6f24d1a07aa2/websockets/chat/src/server.rs#L69

I don't understand what is naïve about the implementation. How would it be improved?

**Comment by robjtede** [comment, marked as answer]
It just means that the chat system itself is not realistic; it's missing many, many features required for real chat apps.
